### ISSUE TITLE
Send reads via COPY TO API with query

### ISSUE THREAD

**Issue by alrocar** [CartoDB/cartoframes#563]
- Stream the result as a dataframe (investigate if it's possible avoiding the file system)

Related CARTOframes methods:
- context.read
- context.query
- layer.QueryLayer (I dont think so, just in case)

**Comment by alrocar** [comment]
Related issues (potentially closable with this one) to be read before starting implement this:

- https://github.com/CartoDB/cartoframes/issues/212

**Comment by oleurud** [comment]
Everything ready with read :rocket: 
context.query will be handled in another task
